src/Components/Web.JS/src/Platform/Circuits/RenderQueue.ts

```diff
@@ -8,6 +8,8 @@ export default class RenderQueue {
 
   private nextBatchId = 2;
 
+  private lastError?: string;
```

Comment:
Consider renaming to `fatalError` since it's not tracking "the most recent" error but rather just one.

Reply:
done